examples/experimental/dagster-airlift/dagster_airlift/core/defs_builders.py

```diff
@@ -38,9 +39,18 @@ def combine_defs(
         elif isinstance(_def, AssetsDefinition):
             assets.append(_def)
         elif isinstance(_def, AssetSpec):
-            assets.append(_def)
+            dag_id = _def.tags[DAG_ID_TAG]
+            task_id = _def.tags[TASK_ID_TAG]
+            specs_by_task_and_dag[(dag_id, task_id)].append(_def)
         else:
             raise Exception(f"Unexpected type: {type(_def)}")
+    for (dag_id, task_id), specs in specs_by_task_and_dag.items():
+
+        @multi_asset(specs=specs, name=f"{dag_id}__{task_id}")
+        def _multi_asset():
+            pass
+
+        assets.append(_multi_asset)
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +29 to +53]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->this seems really strange to include in a generic `combine_defs` function. 

Also why is this necessary? Is the only reason to group these together is because we want to use compute kind?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23846?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
The only reason this is necessary is because we don't currently perform this coercion in build_airflow_defs. But I agree that it's weird, I can just make build_airflow_defs handle coercion of assetspecs (when appropriate).